src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -7117,15 +7143,30 @@ test_client_side_encryption_install (TestSuite *suite)
                "/client_side_encryption/range_explicit_encryption/%s/%s",
                rc.name,
                rangeType);
-            TestSuite_AddFull (
-               suite,
-               test_name,
-               rc.fn,
-               NULL /* dtor */,
-               (void *) rangeTypes[i] /* ctx */,
-               test_framework_skip_if_no_client_side_encryption,
-               test_framework_skip_if_max_wire_version_less_than_19,
-               test_framework_skip_if_single);
+
+            // Skip DecimalNoPrecision if not a replica set.
```

Comment:
Confirming, is the reason for this branch because `test_framework_skip_if_single` allows for sharded clusters, but DecimalNoPrecision requires replica sets only?

Reply:
Yes.